<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepLearningServerPanel</class>
 <widget class="QWidget" name="DeepLearningServerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetDL">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabServer">
      <attribute name="title">
       <string>Deep Learning Server</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>ITK-SNAP Deep Learning Segmentation Extensions</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9" columnstretch="0,1,0,0,0">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <item row="1" column="3">
           <widget class="QToolButton" name="btnEdit">
            <property name="text">
             <string>Edit...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="btnNew">
            <property name="text">
             <string>New...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QToolButton" name="btnDelete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The ITK-SNAP deep learning extension allows you to take advantage of powerful AI-based interactive segmentation tools provided by &lt;a href=&quot;https://github.com/MIC-DKFZ/nnInteractive&quot;&gt;nnInteractive&lt;/a&gt; directly from ITK-SNAP. &lt;/p&gt;
&lt;p&gt;
Please follow &lt;a href=&quot;https://itksnap-dls.readthedocs.io/en/latest&quot;&gt;online documentation &lt;/a&gt; to set up the deep learning extension.
&lt;/p&gt;
&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Server:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="inActiveServer"/>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLabel" name="lblStatus">
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Status:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QToolButton" name="btnReconnect">
            <property name="text">
             <string>Reconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidgetDL</tabstop>
  <tabstop>inActiveServer</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
